cap/modules/workflows/views.py

```diff
@@ -218,19 +198,18 @@ def clone_reana_workflow(workflow_id, workflow=None):
         resp = clone_workflow(workflow_id)
         return jsonify(resp)
     except Exception:
-        return jsonify({
-            'message': ('An exception has occured while retrieving '
-                        'the original workflow attributes.')
-        }), 400
+        raise ReanaWorkflowException(
```

Comment:
whose problem was it? bad request (return info what we sent wrong) or reana server problem (then 503)

Reply:
it's actually a db issue, the cloning just gets the data from our db, no request needed, so in this case...?